components/dashboard/src/teams/OrgSettingsPage.tsx

```diff
@@ -68,27 +72,28 @@ function getTeamSettingsMenu(params: {
     billingMode?: BillingMode;
     ssoEnabled?: boolean;
     orgGitAuthProviders: boolean;
+    isOwner?: boolean;
 }) {
-    const { billingMode, ssoEnabled, orgGitAuthProviders } = params;
+    const { billingMode, ssoEnabled, orgGitAuthProviders, isOwner } = params;
     const result = [
         {
             title: "General",
             link: [`/settings`],
         },
     ];
-    if (ssoEnabled) {
+    if (isOwner && ssoEnabled) {
```

Comment:
We are effectively reimplementing the permissions here in an implicit way. Instead of assuming that owners have access we should rely on FGA to figure that out. Not necessarily as part of this PR, though because it involves more discussion and work.
cc @akosyakov @geropl